css/main.css

```diff
@@ -6,30 +6,57 @@ body{
     background: #2d3436;
 }
 
-.box{
+label {
+    margin-left: 5px;
+}
+
+.bigger {
+    font-size: 26px;
+}
+
+.box {
     width: 300px;
     height: 560px;
     padding: 0px;
     position: absolute;
     top: 50%;
     left: 50%;
-    transform: translate(-50%,-50%);
+    transform: translate(-100%,-50%);
     background: #34495e;
-    text-align: center;
+    text-align: left;
+}
+
+.box-2 {

```

Comment:
Lemme tell you about `display: grid;`. Makes box layouts so nice

Reply:
Planned in future update... maybe